aws/extensions/computetask_awswin_cwlog/extension.ftl

```diff
@@ -0,0 +1,245 @@
+                                r'                            "VERBOSE", ',
+                                r'                            "INFORMATION", ',
+                                r'                            "WARNING", ',
+                                r'                            "ERROR", ',
+                                r'                            "CRITICAL" ',
+                                r'                        ], ',
+                                r'                        "event_name": "Security", ',
+                                r'                        "log_group_name": "' + logGroupName + '", ',
+                                r'                        "log_stream_name": "{instance_id}/security" ',
+                                r'                    } ',
+                                r'                ] ',
+                                r'            } ',
+                                r'        } ',
+                                r'    }, ',
+                                r'    "metrics": { ',
+                                r'        "append_dimensions": { ',
+                                r'            "AutoScalingGroupName": "${aws:AutoScalingGroupName}", ',
+                                r'            "ImageId": "${aws:ImageId}", ',
+                                r'            "InstanceId": "${aws:InstanceId}", ',
+                                r'            "InstanceType": "${aws:InstanceType}" ',
+                                r'        }, ',
+                                r'        "metrics_collected": { ',
+                                r'            "LogicalDisk": { ',
+                                r'                "measurement": [ ',
+                                r'                    "% Free Space" ',
+                                r'                ], ',
+                                r'                "metrics_collection_interval": 60, ',
+                                r'                "resources": [ ',
+                                r'                    "*" ',
+                                r'                ] ',
+                                r'            }, ',
+                                r'            "Memory": { ',
+                                r'                "measurement": [ ',
+                                r'                    "% Committed Bytes In Use" ',
+                                r'                ], ',
+                                r'                "metrics_collection_interval": 60 ',
+                                r'            }, ',
+                                r'            "statsd": { ',
+                                r'                "metrics_aggregation_interval": 60, ',
+                                r'                "metrics_collection_interval": 10, ',
+                                r'                "service_address": ":8125" ',
+                                r'            } ',
```

Comment:
We shouldn't need statsd by default this is a service endpoint that applications can send metrics to. I'd call this an advanced feature and adding it by default might be a bir far

Reply:
Cool